packages/ecommerce-app-base/src/SkuPicker/SkuPicker.tsx

```diff
@@ -154,11 +168,37 @@ export class SkuPicker extends Component<Props, State> {
     }
   };
 
+  constructSearchAdditionalInfo = (
+    makeSearchHelpText: MakeSearchHelpText | undefined,
+    skuType: string | undefined,
+    paginationTotal: number
+  ) => {
+    const helpText = makeSearchHelpText ? makeSearchHelpText(skuType) : '';
+    const totalResults = !!paginationTotal
+      ? `${paginationTotal.toLocaleString()} total results`
```

Comment:
today I learned, `Number.toLocaleString()` super cool.

Reply:
Also what if `paginationTotal` is `0`?  would we still want to render `0 results`?  Or is it ok to render `''`.  Which correct me if I'm misreading this, I think that's what this code would do.

Reply:
Good question! I just checked and the existing behavior is to render `''` instead of `0 results`. I'm just going to keep it as is for now.